tests/test_msg_signtx_grs.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of the TREZOR project.
```

Comment:
I don't see these files in https://github.com/trezor/python-trezor ... are they brand new?

Reply:
Yes, they are based on tests/test_msg_signtx.py from this repo and on test I made for T2 https://github.com/trezor/trezor-core/blob/master/tests/test_apps.wallet.signtx_grs.py

Reply:
ahh